### PR TITLE
testcase for not available yast modules

### PR DESCRIPTION
**Pull request by schubi2**
I have added it to this testcase because it is not worth to generate a single one for it.
Running one testcase takes about 45 min on a slow machine.

### CODE REVIEW

File: spec/tftp.rb [hunk 1]
```diff
@@ -16,6 +16,11 @@
     run_test_script("host.sh")
   end
 
+  # bug 925381
+  it "checks, for unsupported YAST modules" do
```
Comment (kobliha) [inline spec/tftp.rb]: little note: "checks whether unsupported YAST modules are reported" (and maybe even YaST, nitpicking)

File: spec/tftp.xml [hunk 2]
```diff
@@ -77,7 +87,7 @@ chmod 755 /srv/tftpboot
     <errors>
       <log config:type="boolean">true</log>
       <show config:type="boolean">true</show>
-      <timeout config:type="integer">0</timeout>
+      <timeout config:type="integer">10</timeout>
```
Comment (kobliha) [inline spec/tftp.xml]: Maybe comment why?  I know why, but after three months, nobody knows ;)

File: spec/tftp.xml [hunk 3]
```diff
@@ -1,6 +1,16 @@
 <?xml version="1.0"?>
 <!DOCTYPE profile>
 <profile xmlns="http://www.suse.com/1.0/yast2ns" xmlns:config="http://www.suse.com/1.0/configns">
+  <!-- These are not supported anymore -->
```
Comment (kobliha) [inline spec/tftp.xml]: Please reference the bug there too

**Comment by kobliha** [comment]
Although I had a few comments, LGTM, they are minor

File: spec/unsupported_modules.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+set -e -x
+grep "Could not process these unsupported profile sections: \[\"autofs\", \"restore\", \"sshd\"\]" /var/log/YaST2/y2log && echo "AUTOYAST OK"
```
Comment (mvidner) [inline spec/unsupported_modules.sh]: This line is too long

**Pull request merged**